arviz/data/io_tfp.py

```diff
@@ -0,0 +1,49 @@
+"""Tfp-specific conversion code."""
+import numpy as np
+
+from .inference_data import InferenceData
+from .base import dict_to_dataset
+
+
+class TfpConverter:
+    """Encapsulate tfp specific logic."""
+
+    def __init__(self, posterior, *_, var_names=None, coords=None, dims=None):
```

Comment:
I think this just needs to be * to prevent positional arguments. instead of accepting an arbitrary amount

Reply:
Well, I just tried to keep uniform with other converters here.

Reply:
Consistency is good!

The issue here is that this is not consistent. Check out the Pyro implementation
https://github.com/arviz-devs/arviz/blob/master/arviz/data/io_pyro.py#L82

In the current implementation the user can input an arbitrary amount of non keyword arguments, where instead we want to the user to input only one positional argument, and use keywords for the rest. The underscore makes all the difference!

This code snippet might make things more clear.
``` def test_one(posterior, *_, b=2):
    print(posterior)
    print(b)

test_one("posterior", 2, 3, 4, 5, 6, b=10)


def test_two(posterior, *, b=2):
    print(posterior)
    print(b)

test_two("posterior", 2, 3, 4, 5, 6, b=10) 
```

Reply:
oh geez, yes you're right. I got my signals crossed. Ignore this entire chain.